wbooks/src/app/screens/BookList/components/BookItem/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+import noBook from '@assets/img_book_placeholder.png';
+
+import { STYLES } from './styles';
+
+type Props = {
+  item: {
+    title: string;
+    author: string;
+    imageUrl: string | null;
+  };
+};
+
+function BookItem(props: Props) {
```

Comment:
You can destructure props => `function BookItem({ item }: Props) {`
To avoid repeating : `props.prop...`